chopper.py

```diff
@@ -119,6 +142,10 @@ def parse_args() -> dict[str, Any]:
         default=0,
     )
 
+    ## [CG] I had never used `vars` with an argument before, which is super
+    ##      cool! I've found like, `args.input_path` a little more parsimonious
+    ##      than args["input_path"] though, but chalk this up to personal
+    ##      preference.
```

Comment:
See below, but yeah gonna switch to use the Namespace object since it is a little easier on the eyes.